src/app/admin/usersTable/usersTableFilters.ts

```diff
@@ -23,14 +23,14 @@ export const firstNameSearch = (
     query: PostgrestFilterBuilder<Database["public"], any, any>,
     state: string
 ): PostgrestFilterBuilder<Database["public"], any, any> => {
-    return query.ilike("first_name", `%${state}%`);
+    return query.or(`first_name.ilike.%${state}%,first_name.is.null`);
```

Comment:
Does this mean profiles with empty names will always come up even if the filter is set to, say, "abc"?